.github/workflows/_test.yml

```diff
@@ -28,7 +28,12 @@ jobs:
           - "3.9"
           - "3.10"
           - "3.11"
-    name: "make test #${{ matrix.python-version }}"
+        sqlalchemy-version:
```

Comment:
@efriis let me know what you think.

We've had multiple breakages with users using sqlalchemy 1 or 2 for dev work, so adding some support in CI would be useful